.github/workflows/release.yml

```diff
@@ -6,9 +6,6 @@ on:
       - v0.*
       - v1.*
 
-env:
-  CHANNEL: nightly
```

Comment:
Is this intended?

Reply:
I do not see where it is used in https://github.com/timberio/vector/blob/3e4360a2db28b1837e49933e3a6a23b29f97b136/scripts/sync-install.sh, do you know of a reason to keep this?

Reply:
Yup!